students/barecki_andrzej/lesson_06_dicts_tuples_sets_args_kwargs/add_to_inventory.py

```diff
@@ -0,0 +1,22 @@
+def display_inventory(inventory):
+    print("Inventory:")
+    item_total = 0
+    for k, v in inventory.items():
+        item_total += v
+        print(str(v) + ' ' + k)
+    print("Total number of items: " + str(item_total))
+
+
+def add_to_inventory(inventory, added_items):
+    for elem in added_items:
+        if elem not in inventory:
+            inventory[elem] = 1
+        else:
+            inventory[elem] = inventory.get(elem) + 1
+    return inv
```

Comment:
Zwracasz tutaj zmienną `inv`, która nie jest zmienną lokalną. Program działa dlatego, że zaciągnie`inv` z globalnej przestrzeni.

Jest to bug, który ujawni się jeżeli zrobisz kilka ekwipunków lub zmienisz nazwę `inv` na coś innego w linijce 19.